### DISCUSSION TITLE
What is the situation?

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#69] [Q&A]
It does not involve the installation of advertising filter plug -ins, as shown in the instructions of the figure and the login page page, that is, the grains cannot login.URL=http://www.zanli.com 
 The diagram link has been posted:
https://cloud.189.cn/web/share?code=6RVRfiQ3iYz2（访问码：y7yz）
https://cloud.189.cn/web/share?code=vuE3UrN77z2i（访问码：6g31）

**Comment by gz83** [comment]
> It does not involve the installation of advertising filter plug -ins, as shown in the instructions of the figure and the login page page, that is, the grains cannot login.URL=http://www.zanli.com The diagram link has been posted: [https://cloud.189.cn/web/share?code=6RVRfiQ3iYz2（访问码：y7yz）](https://cloud.189.cn/web/share?code=6RVRfiQ3iYz2%EF%BC%88%E8%AE%BF%E9%97%AE%E7%A0%81%EF%BC%9Ay7yz%EF%BC%89) [https://cloud.189.cn/web/share?code=vuE3UrN77z2i（访问码：6g31）](https://cloud.189.cn/web/share?code=vuE3UrN77z2i%EF%BC%88%E8%AE%BF%E9%97%AE%E7%A0%81%EF%BC%9A6g31%EF%BC%89)

@whxuezhe Hello, does Chromium browser have this problem?

**Reply by whxuezhe** [reply]
@gz83 Chromium & Opera did not reproduce this problem,and I have found new problems. In the Thorium browser, it is impossible to log in to Mail.163.com and cannot log in to the PC's mailbox web page through the Code Master APP.
Firefox browser logs in to the NetEase mailbox is normal, and these browsers of QQ mailboxes cannot enter the account password to log in, but can only scan the code.
Interture an other concern problem, I know that the construction will not make an extremely unstable half -product, which will be published as an OPERA. I also know that the CI test encounters due to frequent changes in the construction of 110.0.111.0 follow -up. When Frack and the AVX, AVX2 instruction set the special version, they encountered severe bugs such as collapse. In the early days, I always thought that I could follow the Google Golden Silk in this way. AVX2 followed up the packaging version of the development team from chromium.woolyss.com J. ; This method of applying may not be applicable. In addition, I asked the proposal to recruit internal test group members for the Thorium browser, did it be submitted to Alex? Can there be approval and final finalization? Essence

**Reply by gz83** [reply]
> @gz83 Chromium & Opera did not reproduce this problem,and I have found new problems. In the Thorium browser, it is impossible to log in to Mail.163.com and cannot log in to the PC's mailbox web page through the Code Master APP.

@whxuezhe Please back up the current Thorium user data, then delete the Thorium user data folder, and test again under clean user data

You can pay attention to woolyss and github repo for Thorium's release information, and we do not set up the so-called internal test group or internal members

**Reply by whxuezhe** [reply]
@gz83 Thank you for your patience. I found something new and needed to be improved.
Contact with CCLEANER to achieve the purpose of cache cleaning of Thorium browser; and contact with Avast Antitrack to counter tracking, as adaptive, and the protection support of the Thorium browser. Thank you for your unremitting efforts!

**Reply by gz83** [reply]
> @gz83 Thank you for your patience. I found something new and needed to be improved. Contact with CCLEANER to achieve the purpose of cache cleaning of Thorium browser; and contact with Avast Antitrack to counter tracking, as adaptive, and the protection support of the Thorium browser. Thank you for your unremitting efforts!

@whxuezhe Your comments have been received and we will evaluate them, but please note that the proposed suggestions or functions may not necessarily be implemented.

**Reply by whxuezhe** [reply]
@gz83 I feel that the method you say is not feasible. It is recommended that you check the elements and Devtool to see the web source code and the relevant API interface, and I also discovered:https://www.ysense.com Its login requests the RECAPCHA plug -in to verify the login password through the reverse proxy request. Only the earlier Chromium GitHub version has been stable, and the stability of Google Golden Silk is relatively poor and occasionally cannot log in. In addition, since we have Google's Sync Sandbox; I hope the next version will be constructed and update the Sandbox V4 setting and V4 version of UI; thank you!

**Reply by gz83** [reply]
> @gz83 I feel that the method you say is not feasible. It is recommended that you check the elements and Devtool to see the web source code and the relevant API interface, and I also discovered:https://www.ysense.com Its login requests the RECAPCHA plug -in to verify the login password through the reverse proxy request. Only the earlier Chromium GitHub version has been stable, and the stability of Google Golden Silk is relatively poor and occasionally cannot log in. In addition, since we have Google's Sync Sandbox; I hope the next version will be constructed and update the Sandbox V4 setting and V4 version of UI; thank you!

@whxuezhe In the future, when you reply to our comments or make suggestions, please try to use relatively standard English expressions, and please do not mix too much irrelevant content in your reply.

I know that not all users who use Thorium speak English as their first language, we allow you to use the translation tool to communicate with us, but please double check the content when translating.

If the RECAPCHA verification code and the website you mentioned appear abnormal on your side, please carefully check the computer environment and network environment (especially the network, whether there are restrictions), and please use multiple different browsers  Test, if there are abnormalities in multiple different browsers, it is likely to be a problem with the relevant website.

Thorium and Chrome Canary are compiled and developed based on very early development versions. There will definitely be some bugs or performance regressions (visible or invisible to the naked eye). If you cannot accept bugs or performance regressions, please do not use these versions.  This can also affect your experience with your computer.

Regarding Thorium's feature request, I have already answered you in the previous comment, and I will not repeat the content of the previous comment.

Finally, please maintain respect for Thorium's development team members and other users, pay attention to the wording of comments, and recommend you to check out the COC of our repo: https://thorium.rocks/COC.html

**Reply by whxuezhe** [reply]
@gz83 First of all, I apologized to the development team members! I can accept some non -serious bugs. Please understand that I am currently a software evaluation enthusiast in the junior and junior and junior and junior exploration phases. At all, I have not thoroughly understood the development process and other information. It is also due to some described organic diseases. I can't fulfill my studies and ideals (once fantasized to become China's Bill Gates) according to my hobbies. These are currently fantasy bubbles. At least I found that we are responsible for fixing some bugs. Some of them mentioned that they knew but did not fix, or even refused to recognize the defects. Microsoft's third brother was even more strange!
